fed/mosaic-application/src/main/java/org/eclipse/mosaic/fed/application/ambassador/navigation/IRoutingModule.java

```diff
@@ -69,4 +69,14 @@ public interface IRoutingModule {
      * @return The road position, which is closest to the given location.
      */
     IRoadPosition getClosestRoadPosition(GeoPoint geoPoint);
+
+    /**
+     * Returns the road position, which is closest to the given {@link GeoPoint}.
+     * If two adjacent edges overlap, the heading will be used as a similarity measure.
+     *
+     * @param geoPoint The geographical location to search a road position for.
+     * @param heading  used as a measure of similarity if multiple edges match
+     * @return The road position, which is closest to the given location.
+     */
+    IRoadPosition getClosestRoadPosition(GeoPoint geoPoint, double heading);
```

Comment:
Should be added to `INavigationModule` as well

Reply:
whoops, will do!

Reply:
Though this might be somewhat unnecessary as the `NavigationModule` (which is currently the only implementation of `INavigationModule`) implements both the `IRoutingModule` and the `INavigationModule`

Reply:
the `getNavigationModule()` method returns a `INavationModule`. 

NavigationModule implements both interfaces so we dont need seperate implementations for IRoutingModule and INavigationModule.